### ISSUE TITLE
Enter the Whistle Number

### ISSUE THREAD

**Issue by shiffman** [CodingTrain/Random-Whistle#34]
@jasontheillustrator and I were chatting about the page yesterday and it's going to prove too complex to custom print a different QRCode for each whistle. Instead since each whistle is etched with a unique number, that number can be entered and then the custom message with random walk would appear afterwards.

Jason, would you have time to mock up a design on how this might work and then we'll implement?

This is the last step I think!

@KobeLiesenborgs I think we can start with a simple JSON file and decide later if we want to hide the data in cloud storage.

**Comment by KobeLiesenborgs** [comment]
Sure thing! Let me know once you have a design! Reading from a JSON should be trivial! How do we want to go about generating the name/number pairs? Is there data available?

**Comment by jasontheillustrator** [comment]
Sorry for the delay. I actually posted these in the other thread, but should have posted them here.
![Whistle-Webpage-Mockup-2-Horizontal](https://user-images.githubusercontent.com/103963888/214100114-0cb4a548-b591-4bd3-8626-476c6463f7b9.jpg)
![Whistle-Webpage-Mockup-2-Vertical-Download](https://user-images.githubusercontent.com/103963888/214100118-4c2726a9-5647-44ac-9248-c950c2618927.jpg)
![Whistle-Webpage-Mockup-2-Vertical](https://user-images.githubusercontent.com/103963888/214100122-86142728-7252-41db-b993-897619316847.jpg)

**Comment by jasontheillustrator** [comment]
I made 3 layouts. There is the horizontal layout which modifies what has currently been done to include an input and button. There's a vertical version that is more mobile friendly. There's a vertical version that includes a download button at the bottom in case someone wants to download the randomwalk as a phone background. Also, for all 3 layouts I adjusted the header graphic so the characters are bigger.
